hubble-be/src/main/java/com/baidu/hugegraph/controller/GraphConnectionController.java

```diff
@@ -165,21 +165,20 @@ public GraphConnection update(@PathVariable("id") int id,
         if (oldEntity == null) {
             throw new ExternalException("graph-connection.not-exist.id", id);
         }
-        GraphConnection entity = this.mergeEntity(oldEntity, newEntity);
         // Make sure the updated connection doesn't conflict with exists
-        this.checkEntityUnique(entity, false);
-        entity = sslService.configSSL(this.config, entity);
-        HugeClient client = HugeClientUtil.tryConnect(entity);
+        this.checkEntityUnique(newEntity, false);
+        this.sslService.configSSL(this.config, newEntity);
+        HugeClient client = HugeClientUtil.tryConnect(newEntity);
         // Check current graph's data size
         LicenseService.VerifyResult verifyResult;
         verifyResult = this.licenseService.verifyDataSize(
-                       client, entity.getName(), entity.getGraph());
+                       client, newEntity.getName(), newEntity.getGraph());
         Ex.check(verifyResult.isEnabled(), Constant.STATUS_UNAUTHORIZED,
                  verifyResult.getMessage());
 
-        this.connService.update(entity);
-        this.poolService.put(entity, client);
-        return entity;
+        this.connService.update(newEntity);
+        this.poolService.put(newEntity, client);
+        return newEntity;
```

Comment:
It need not return newEntity since the returned object is the parameter

Reply:
It is indeed worth considering, but now the front end relies on this return result, if modification will cause the front end to also need to be changed